### DISCUSSION TITLE
RabbitMQ no longer running after Haswell EVC to Saphire EVC Compatible Mode change.  Is this expected?

### DISCUSSION THREAD

**Discussion by ByrdsongSpok** [rabbitmq/rabbitmq-server#12495] [Questions]
### Community Support Policy

- [X] I have read [RabbitMQ's Community Support Policy](https://github.com/rabbitmq/rabbitmq-server/blob/main/COMMUNITY_SUPPORT.md)
- [X] I agree to provide all relevant information (versions, logs, rabbitmq-diagnostics output, detailed reproduction steps)

### RabbitMQ version used

3.12.x or older

### Erlang version used

26.0.x

### Operating system (distribution) used

Windows Server 2016

### How is RabbitMQ deployed?

Windows installer

### rabbitmq-diagnostics status output

See https://www.rabbitmq.com/docs/cli to learn how to use rabbitmq-diagnostics
<details>

```
# PASTE OUTPUT HERE, BETWEEN BACKTICKS
```
</details>

### Logs from node 1 (with sensitive values edited out)

See https://www.rabbitmq.com/docs/logging to learn how to collect logs
<details>

```
# PASTE LOG HERE, BETWEEN BACKTICKS
```
</details>

### Logs from node 2 (if applicable, with sensitive values edited out)

See https://www.rabbitmq.com/docs/logging to learn how to collect logs
<details>

```
# PASTE LOG HERE, BETWEEN BACKTICKS
```
</details>

### Logs from node 3 (if applicable, with sensitive values edited out)

See https://www.rabbitmq.com/docs/logging to learn how to collect logs
<details>

```
# PASTE LOG HERE, BETWEEN BACKTICKS
```
</details>

### rabbitmq.conf

See https://www.rabbitmq.com/docs/configure#config-location to learn how to find rabbitmq.conf file location
<details>

```
# PASTE rabbitmq.conf HERE, BETWEEN BACKTICKS
```
</details>

### Steps to deploy RabbitMQ cluster

TBD

### Steps to reproduce the behavior in question

TBD

### advanced.config

See https://www.rabbitmq.com/docs/configure#config-location to learn how to find advanced.config file location
<details>

```
# PASTE advanced.config HERE, BETWEEN BACKTICKS
```
</details>

### Application code

<details>

```python
# PASTE CODE HERE, BETWEEN BACKTICKS
```
</details>

### Kubernetes deployment file

<details>

```yaml
# Relevant parts of K8S deployment that demonstrate how RabbitMQ is deployed
# PASTE YAML HERE, BETWEEN BACKTICKS
```
</details>

**Comment by ggallagher-spok** [comment]
### Windows Configuration
It documents known conditions and scenarios which can cause RabbitMQ Windows service or CLI tools to malfunction.
https://www.rabbitmq.com/docs/windows-configuration

pulled from a group thread that identified a known issue with Windows Services:

Oct 31, 2015, 1:24:45 AM
to rabbitm...@googlegroups.com, SallySue
 On 31 October 2015 at 00:45:12, SallySue ([jennifer...@pearson.com](https://groups.google.com/)) wrote:
> Hello. I just wanted to note this for people looking for a solution.
> Essentially, I reinstalled RabbitMQ Server and changed the
> directory to eliminate all spaces and characters, set to D:\RabbitMQServer,
> all started well, I have my logs now in the directory I defined
> below and was able to start and have the service stay running under
> the domain account.

while we understand that this is not great, this is a known Windows-specific issue
that has been documented for a long time:
https://www.rabbitmq.com/windows-quirks.html
--
MK

Staff Software Engineer, Pivotal/RabbitMQ

**Comment by michaelklishin** [comment]
RabbitMQ 3.12 is [completely out of support](https://www.rabbitmq.com/release-information).

Even if it wasn't, you have provided no details whatsoever for our team to work with. Presumably an upgrade of sorts has taken place and… we should guess the rest? 

Perhaps the Windows service is not started on VM boot. We do not guess in this community.
Start with inspecting [server logs](https://www.rabbitmq.com/docs/logging).
